app/src/main/java/com/heapixLearn/discovery/domain/contact/DumbDBContactManager.java

```diff
@@ -0,0 +1,51 @@
+package com.heapixLearn.discovery.domain.contact;
+
+import com.heapixLearn.discovery.logic.contact.LogicContact;
+import com.heapixLearn.discovery.logic.contact.contacts_domain_interfaces.DBStoreContactManager;
+
+import java.util.ArrayList;
+import java.util.concurrent.FutureTask;
+
+public class DumbDBContactManager implements DBStoreContactManager {
```

Comment:
@dianaprimshits unused class, should be removed

Reply:
@YodaMike Fixed